### DISCUSSION TITLE
Manim v0.18.0

### DISCUSSION THREAD

**Discussion by behackl** [ManimCommunity/manim#3447] [Announcements]
The full changelog can be found at https://docs.manim.community/en/latest/changelog/0.18.0-changelog.html.

<hr /><em>This discussion was created from the release <a href='https://github.com/ManimCommunity/manim/releases/tag/v0.18.0'>Manim v0.18.0</a>.</em>

**Comment by ubaldot** [comment]
While working with Manim this morning, my terminal windows displayed that a new version of Manim was available! Such a great news! I will install it as soon as it will be available with conda. :) 

However, it looks like that the  html.zip file related to the docs is missing. 
Is the action for building it still running (I know it takes time!)? 

Anyway, thanks to all the contributors!

**Reply by behackl** [reply]
You've just won he award for finding the first new bug in the new release! 🏆 

The action ran successfully, but due to a path problem the zipped file was not correctly added to the release. I'll look into it. :-)

**Reply by ubaldot** [reply]
Ahahahaha!!!
Wow, I am honored 😅 
Sorry I didn’t want to be the party spoiler 😕

Anyway, as you are already checking that, could you please also add the v18.0 also to conda-forge? 😇😁

**Reply by behackl** [reply]
A .zip of our documentation has been added to the release page. The maintainer of our conda package is aware of the situation, it should be updated soon. 👍

**Reply by ubaldot** [reply]
hum... sorry to say but... it looks like there is some issue with the docs.
Here is how index.html renders on my mac: 

<img width="1119" alt="image" src="https://github.com/ManimCommunity/manim/assets/10845685/5da700ff-e252-4732-83e9-354faf702483">

**Reply by behackl** [reply]
It appears something went wrong when I packaged it. I've just rebuilt and repackaged the docs locally, the new version should work as expected.

The file size is actually somewhat larger than I would have expected, we should probably investigate. :thinking:

**Reply by ubaldot** [reply]
This works well, but perhaps the index.html should be fixed? It says v17.3.  
Thank you for your effort in doing this! :) This feature is very valuable!

![image](https://github.com/ManimCommunity/manim/assets/10845685/9258256f-4bd2-4dcd-8ce8-037595e4694e)

**Reply by behackl** [reply]
... fixed that too now. :smile:
